BoxContentSDK/BoxContentSDK/Operations/BOXStreamOperation.h

```diff
@@ -14,12 +14,10 @@
 typedef void (^BOXAPIStreamProgressBlock)(NSData *data, long long expectedTotalBytes);
 
 /**
- * BOXAPIDataOperation is a concrete subclass of BOXAPIAuthenticatedOperation.
- * This operation receives binary data from the Box API which may be in the form
- * of downloads or thumbnails.
+ * BOXStreamOperation is a concrete subclass of BOXAPIAuthenticatedOperation.
```

Comment:
Lots of this header were apparently copy/pasted years ago. Trying to clean it up a bit.